examples/scripts/configure-solr.sh

```diff
@@ -7,29 +7,27 @@
 # `solr` in the `installed_extras`, and is currently set up for the D7 versions
 # of Apache Solr Search or Search API Solr.
 
-SOLR_SETUP_COMPLETE_FILE=/etc/drupal_vm_solr_config_complete
+NAME="drupalvm"
```

Comment:
I second that motion! I didn't know there is a default provided in Search API Solr Search because I am currently using Apache Solr Search. Let me do the change.